extensions/mssql/src/objectExplorerNodeProvider/objectExplorerNodeProvider.ts

```diff
@@ -107,16 +108,31 @@ export class MssqlObjectExplorerNodeProvider extends ProviderBase implements azd
 			if (node) {
 				expandResult.errorMessage = node.getNodeInfo().errorMessage;
 				let children = await node.getChildren(true);
-				if (children) {
-					expandResult.nodes = children.map(c => c.getNodeInfo());
-					// There is only child returned when failure happens
-					if (children.length === 1) {
-						let child = children[0].getNodeInfo();
-						if (child && child.nodeType === constants.MssqlClusterItems.Error) {
-							expandResult.errorMessage = child.label;
-							expandResult.nodes = [];
+				if(children && children.length > 0)
+				{
```

Comment:
Styling: for Typescript we put the `{` on the same line as the `if` and `else` statements. I think running the formatter on the doc will fix these all up.